lib/ftw/connection.rb

```diff
@@ -349,7 +351,11 @@ def secure(options=nil)
     sslcontext = OpenSSL::SSL::SSLContext.new
     # If you use VERIFY_NONE, you are removing the trust feature of TLS. Don't do that.
     # Encryption without trust means you don't know who you are talking to.
-    sslcontext.verify_mode = OpenSSL::SSL::VERIFY_PEER
+    if settings[:ssl_verify]
```

Comment:
Would you prefer a new call back altogether, something like:
```ruby
def ignore_certificate()
  return true
end
```
or would adding logic to certificate_verify() be sufficient?